modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-creating-ostree-iso_{context}"]
+= Creating the {op-system-ostree-first} image
+
+Use the following procedure to create the ISO. The {op-system-ostree} Installer image pulls the commit from the running container and creates an installable boot ISO with a Kickstart file configured to use the embedded OSTree commit.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root-user access to your build host.
+* You have the `podman` tool.
+
+.Procedure
+
+. Start an `ostree` container image build by running the following command:
++
+[source,terminal]
+----
+$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/9/$(uname -m)/edge" minimal-microshift edge-container | awk '{print $2}')
+----
++
+This command also returns the identification (ID) of the build for monitoring.
+
+. You can check the status of the build periodically by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli compose status
+----
++
+.Example output of a running build
+
+[source,terminal]
+----
+ID                                     Status     Time                      Blueprint            Version   Type               Size
+cc3377ec-4643-4483-b0e7-6b0ad0ae6332   RUNNING    Wed Jun 7 12:26:23 2023   minimal-microshift   0.0.1     edge-container
+----
++
+.Example output of a completed build
+
+[source,terminal]
+----
+ID                                     Status     Time                      Blueprint            Version   Type               Size
+cc3377ec-4643-4483-b0e7-6b0ad0ae6332   FINISHED   Wed Jun 7 12:32:37 2023   minimal-microshift   0.0.1     edge-container
+----
+
+[NOTE]
+====
+You can use the `watch` command to monitor your build if you are familiar with how to start and stop it.
+====
+
+. Download the container image using the ID and get the image ready for use by running the following commands:
++
+[source,terminal]
+----
+$ sudo composer-cli compose image ${BUILDID}
+----
+
+. Change the ownership of the downloaded container image to the current user by running the following command:
++
+[source,terminal]
+----
+$ sudo chown $(whoami). ${BUILDID}-container.tar
+----
+
+. Add current-user read permissions to the image by running the following command:
```

Comment:
Maybe reword to avoid the compound adjective here for clarity?
```suggestion
. Add read permissions for the current user to the image by running the following command:
```